xla/stream_executor/rocm/rocm_platform.h

```diff
@@ -63,7 +63,7 @@ class ROCmPlatform : public Platform {
   // looking in or storing to the Platform's executor cache.
   // Ownership IS transferred to the caller.
   absl::StatusOr<std::unique_ptr<StreamExecutor>> GetUncachedExecutor(
-      const StreamExecutorConfig& config) override;
```

Comment:
Is this related?

Reply:
Oh, it's a fix for an unrelated breakage. I see.

Reply:
FYI, this fix is being submitted separately in https://github.com/openxla/xla/pull/15726.